src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditEntityExactlyOne.razor.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class EditEntityExactlyOne : ComponentBase
```

Comment:
I would call this component Select or SelectList. It has nothing about 'edit' - if it is a dropdown it already means we're editing something. It has nothing about 'ExactlyOne', as we can select zero or one. 
In the future the component for multiple selection can be called SelectMany or SelectManyList.

Reply:
within latest commit